app/models/product_import/unit_converter.rb

```diff
@@ -32,6 +32,8 @@ def unit_scales
       {
         'g' => { scale: 1, unit: 'weight' },
         'kg' => { scale: 1000, unit: 'weight' },
+        'oz' => { scale: 28.35, unit: 'weight' },
+        'lb' => { scale: 453.6, unit: 'weight' },
```

Comment:
this code is repeated from  units option_value_namer#scale_for_unit_value
this should be extracted to a service called Units that would contain these data structured with these values and strings.

Reply:
this is an extra improvement on top of my PR. We can skip this for now.

Reply:
I was thinking the same, we could really use a refactoring into a single place for Units to live, and ideally have that be available to both the front end javascript/coffeescript files as well as the rails backend... but agreed it can be a separate effort.